account_analytic_default_account/tests/test_analytic_default_account.py

```diff
@@ -141,3 +152,86 @@ def test_account_analytic_default_account_move(self):
         self.assertEqual(move_line_1.analytic_account_id,
                          self.analytic_account_4)
         self.assertFalse(move_line_2.analytic_account_id.id)
+
+        # set company
+        self.account_analytic_default_model.create({
+            'account_id': self.account_sales.id,
+            'analytic_id': self.analytic_account_1.id,
```

Comment:
IMO would be better to create one `account.analytic.account` for each `account.analytic.default` that you use in the test (as is done there https://github.com/OCA/account-analytic/pull/237/files#diff-467b4b47594859ebd20aea47ff40cf36R58 ), just to make sure the default record that is used is the one you created and not an existing one created in the `setUpClass`.

Reply:
@grindtildeath updated